### ISSUE TITLE
Terrain tiles perspective merging

### ISSUE THREAD

**Issue by franciscod** [SFTtech/openage#141]
Let's say i have the following terrain texture made by A Bad Guy (notice the colour dots in the alpha space)
![15001 slp](https://cloud.githubusercontent.com/assets/726447/4948913/efaef48a-6627-11e4-9748-1fe34d8216e6.png)

The alpha space is outside the rhombus, so it'd be okay... [_right?_](https://www.youtube.com/watch?v=YztLZ_NfgdQ)

![1415339142](https://cloud.githubusercontent.com/assets/726447/4948925/367f1084-6628-11e4-9735-706f281df0ad.png)

NOPE! THE DOTS TRY TO REACH GABEN!

Basically each tile wastes half the space.

We should fix this :grinning:

**Comment by TheJJ** [comment]
We could do perspective rendering of the terrain in theory with the new renderer (#287), but i'll try to implement the single-quad idea. Otherwise we'd have to transform the original terrain tiles to be non-perspective to render them perspectively again later.
Perspective rendering would be required though once aoe:hd removes the legacy files and just ships the terrain textures directly (which will happen one day..)

So I'll prepare perspective terrain rendering (but for that the coordinate system has to be improved..) but implement the above squaring of the terrain tiles.

**Comment by heinezen** [comment]
Implemented in https://github.com/SFTtech/openage/pull/1497
